### PR TITLE
Update UT3Paladin.uc

### PR DESCRIPTION
**Pull request by HellDragon-HK**
Re-try

### CODE REVIEW

**Comment by GreatEmerald** [comment]
Does this also require binary updates?

**Comment by HellDragon-HK** [comment]
Very much so because the suspension fix lowered it the ground so much it needs a collision box edit

File: Classes/UT3Paladin.uc [hunk 1]
```diff
@@ -80,13 +95,13 @@ defaultproperties
...
-    ExplosionSounds(2)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode01';
-    ExplosionSounds(3)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode01';
-    ExplosionSounds(4)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode01';
+    DamagedEffectHealthFireFactor=0.40 //0.25
+    DamagedEffectFireDamagePerSec=2.00 //0.75
+    ExplosionSounds(0)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode02;
+    ExplosionSounds(1)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode02;
+    ExplosionSounds(2)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode02;
+    ExplosionSounds(3)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode02;
+    ExplosionSounds(4)=Sound'UT3A_Vehicle_Paladin.Sounds.A_Vehicle_Paladin_Explode02;
```
Comment (GreatEmerald) [inline Classes/UT3Paladin.uc]: These should be made into a single array as well, with `ExplosionSounds=()` above.

**Comment by GreatEmerald** [comment]
OK, thanks.

**Pull request merged**